src/NSwag.CodeGeneration.TypeScript/Templates/Client.ProcessResponse.HandleStatusCode.liquid

```diff
@@ -90,7 +90,7 @@ return{% if operation.HasResultType %} null{% endif %};
 {%             endif -%}
 {%         endif -%}
 {%     else -%}
-return throwException({% if Framework.IsAngularJS %}this.q, {% endif %}"A server error occurred.", status, _responseText, _headers);
+return throwException({% if Framework.IsAngularJS %}this.q, {% endif %}"{{ response.ExceptionDescription }}", status, _responseText, _headers);
```

Comment:
Is `ExceptionDescription` an existing property on the repsonse model?

Reply:
Maybe we should fall back to the current message if the description is null or empty?

Reply:
Apologies, I should have put more info in the summary. `ExceptionDescription` is an existing property and never returns `null` or empty string.

https://github.com/RicoSuter/NSwag/blob/85ae862fd6d68173a201a79e0ad06e0be2ec5de1/src/NSwag.CodeGeneration/Models/ResponseModelBase.cs#L94-L96